sdk/keyvault/azure-keyvault-securitydomain/azure/keyvault/securitydomain/operations/__init__.py

```diff
@@ -0,0 +1,19 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) Python Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+from ._operations import HsmSecurityDomainOperations
```

Comment:
The `HsmSecurityDomainOperations` class is being exposed as a public API and shows up in the APIView. This intentional?

Reply:
Hm, that's unintentional. I can't seem to target an operation interface with `@@access` to make it private in our `client.tsp`, so I'll need to figure out how to prevent the emitter from exposing this class